docs/docs/other-css-frameworks.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Other CSS Frameworks
+---
+
+This guide introduces some other popular styling libraries that you might want to use in your project.
+
+## How to choose a framework
+
+There's no one right answer for which framework you should use. Ultimately it's a matter of personal preference and what works best for your project.
+
+Here are some things you might want to consider when choosing a styling framework:
+
+- Is it [accessible](/docs/making-your-site-accessible/)?
+- Is it responsive? How will it look on a mobile device?
+- Does it require additional dependencies?
+- Does it have good documentation?
+- Are there any browsers it doesn't support?
+
+## Bootstrap
+
+Bootstrap is a widely used CSS and JS framework that was created at Twitter in 2010. It gained popularity for its support of responsive and mobile-first design.
+
+Under the hood, Bootstrap uses [Sass](https://sass-lang.com/). Unlike other frameworks in this guide, Bootstrap does not provide React components directly. Instead, it provides a set of pre-built CSS class names like `btn-primary` or `dropdown` that you can use to style HTML elements.
+
+If you're using Bootstrap v4 or older, you'll also need to include jQuery to get many of the JavaScript components to work correctly. As of [Bootstrap v5](https://blog.getbootstrap.com/2020/06/16/bootstrap-5-alpha/), the framework no longer depends on jQuery.
+
+**Browser-support warning:** Bootstrap v5 also dropped Internet Explorer support.
```

Comment:
```suggestion
**Note:** Bootstrap v5 also dropped Internet Explorer support.
```

I know this is less specific but it's also more consistent with similar notes throughout the docs.

Reply:
Sounds good, I'm def pro-consistency :)